post-office-stub/src/data/postOfficeResponse/postResponse400.ts

```diff
@@ -0,0 +1,9 @@
+export const POST_REPONSE_400 = {
+	"errors": [
+		{
+			"error": "BadRequestError",
+			"message": "Can't send to this recipient using a team-only API key",
```

Comment:
Are these being used if so can we change the message please these ones are for GovNotify